vm/chef/cookbooks/elasticsearch/files/elasticsearch

```diff
@@ -42,6 +49,12 @@ readonly total_memory_kb="$(awk '/MemTotal/ { print int($2) }' /proc/meminfo)"
 readonly total_memory_gb="$((total_memory_kb / 1024 / 1024))"
 readonly es_heap_size_gb="$((total_memory_gb / 2))"
 
+# Set protocol as https if configuration is enabled
+es_protocol="http"
+if [[ "${es_use_https:-"false"}" == "true" ]]; then
```

Comment:
this can can be moved to following section ->
```
# Install and configure self-signed certificates if option is selected
```

to keep all configuration for https together